CalculatorApp/CalculatorApp/Program.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Reflection.Emit;
+
+namespace CalculatorApp
+{
+
+    internal class Program
+    {
+        enum Menu { Sum = 1, Subtraction = 2, Division = 3, Multiplication = 4, Power = 5, Root = 6, Exit = 7 }
+
+        static void Main(string[] args)
+        {
+            bool choosedExit = false;
+            while (!choosedExit) // While the user DO NOT(!) choose Exit, show the Menu.
+            {
+                Console.WriteLine("Welcome to the Calculator App! Please, select one of the options:");
+                Console.WriteLine("1-Sum\n2-Subtraction\n3-Division\n4-Multiplication\n5-Power\n6-Root\n7-Exit");
+
+                /*
+                string optionsTXT = Console.ReadLine();
+                int optionsInt = int.Parse(optionsTXT);
+                Menu options = (Menu)optionsInt;
+                */
+
+                Menu options = (Menu)int.Parse(Console.ReadLine());
```

Comment:
This parse is unsafe. It would throw an exception for non numeric user inputs. Please explore int.TryParse() method and apply the changes. In case of non numeric user input the app should warn the user and continue with waiting for menu item selection.